### DISCUSSION TITLE
now available on conda-forge

### DISCUSSION THREAD

**Discussion by kthyng** [podaac/data-subscriber#58] [Announcements]
Hi! I took the liberty of putting this package on conda-forge:

https://anaconda.org/conda-forge/podaac-data-subscriber

I hope someone else finds it helpful too!

**Comment by mike-gangl** [comment]
Very cool! Thank you. Do you have any instructions on how we can make the publishing to Anaconda a part of our release process? I've never done it before, but would be interested in automating it and keeping it up to date.

**Comment by kthyng** [comment]
Hi! Yes here are some notes I have to update a package:
1. Follow directions here: https://conda-forge.org/docs/maintainer/updating_pkgs.html#example-workflow-for-updating-a-package
2. Go to your feedstock: https://github.com/conda-forge/podaac-data-subscriber-feedstock
3. fork to your personal or professional github account (something where you have control)
4. clone locally
5. edit the recipe/meta.yaml file, in particular pay attention to:
- the version at the top
- the hash
- if you have a PyPI recipe (which you do), you can run python package grayskull to get the updated meta.yaml file and then use that in your feedstock. (this is the easiest way I've found to update)

I think there may be ways to automate this — I've tried with Github actions but haven't gotten it to work so far.
